### ISSUE TITLE
Swagger API: type error handling a query parameter with csv format

### ISSUE THREAD

**Issue by seanpk** [BlueOakJS/blueoak-server#55]
If you don't provide this optional query parameter, the swagger validor 
Here's the stack:
```
TypeError: str.split is not a function
    at parseArray (/tmp/my-project/node_modules/blueoak-server/lib/swaggerUtil.js:74:16)
    at Object.castValueFromString [as cast] (/tmp/my-project/node_modules/blueoak-server/lib/swaggerUtil.js:48:17)
    at setDefaultQueryParams (/tmp/my-project/node_modules/blueoak-server/handlers/_swagger.js:231:52)
    at /tmp/my-project/node_modules/blueoak-server/handlers/_swagger.js:197:13
    at validateParameters (/tmp/my-project/node_modules/blueoak-server/handlers/_swagger.js:345:12)
    at /tmp/my-project/node_modules/blueoak-server/handlers/_swagger.js:193:9
    at Layer.handle [as handle_request] (/tmp/my-project/node_modules/express/lib/router/layer.js:95:5)
    at next (/tmp/my-project/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/tmp/my-project/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/tmp/my-project/node_modules/express/lib/router/layer.js:95:5)
    at /tmp/my-project/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/tmp/my-project/node_modules/express/lib/router/index.js:330:12)
    at next (/tmp/my-project/node_modules/express/lib/router/index.js:271:10)
    at /tmp/my-project/node_modules/i18next/lib/i18next.js:119:13
    at /tmp/my-project/node_modules/i18next-client/i18next.js:992:21
    at /tmp/my-project/node_modules/i18next/lib/i18nextWrapper.js:317:29
    at /tmp/my-project/node_modules/i18next/lib/i18nextWrapper.js:360:33
    at ReadFileContext.callback (/tmp/my-project/node_modules/i18next/lib/filesync.js:19:17)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:303:13)
```

Here's an example query definition:
```yaml
name: types
in: query
required: false
type: array
items:
  type: string
  enum:
    - ANY
    - Type_A
    - Type_R
    - Type_S
collectionFormat: csv
default:
  - ANY
description: |
  A comma-separated list of types to search against , or the value `ANY` for all.
```

**Comment by pwwolf** [comment]
Looking at this now. More specifically it occurs when a default is specified, and the default is an array object.

**Comment by pwwolf** [comment]
Fixed in 2.2.5 3e30e3399103e1876189da13a701798f55ee6869
